app/views/_layouts/content-page.nunjucks

```diff
@@ -22,7 +22,7 @@
           {% endif %}
         </div>
 
-        <div class="column__one-third column__right">
+        <div class="column__one-third-right">
```

Comment:
Moved the `.column__right` class into a full set of `xxx-xxx-right` classes, as per the grid mixins. Mainly so there's less multiple reference to breakpoints.